Tools/bump-versions.py

```
@@ -0,0 +1,90 @@
+    component = sys.argv[1]
+
+    manifests = sorted(APPS_DIR.glob(f"*/{MANIFEST}"))
+    if not manifests:
+        print(f"No manifests found in {APPS_DIR}", file=sys.stderr)
+        return 1
+
+    print(f"Bumping {component} (+ version.code) in {len(manifests)} apps:\n")
+    for manifest in manifests:
+        process_manifest(manifest, component)
+
+    print(f"\nDone.")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)